lib/rucaptcha/engine.rb

```diff
@@ -6,7 +6,7 @@ class Engine < ::Rails::Engine
       # https://github.com/rails/rails/blob/3-2-stable/actionpack/lib/action_dispatch/routing/route_set.rb#L268
       # `app.routes.prepend` start from Rails 3.2 - 5.0
       app.routes.prepend do
-        mount RuCaptcha::Engine => "/rucaptcha"
+        mount RuCaptcha::Engine => RuCaptcha.config.mount_path || "/rucaptcha"
```

Comment:
Move default value initialize to `lib/rucaptcha.rb` same like other config keys.